Validation/HGCalValidation/test/python/runHFNoseDigiStudy_cfg.py

```diff
@@ -19,7 +18,7 @@
     process.MessageLogger.HGCalGeom=dict()
 
 process.source = cms.Source("PoolSource",
-                            fileNames = cms.untracked.vstring('file:step2.root')
+                            fileNames = cms.untracked.vstring('file:step2D94.root')
                             )
```

Comment:
Would it be useful to add the same flexibility of handling the geometry via a command line parameter?
Probab;y `D94` won't be the last geometry we will use for `HFNose`.